assets/scss/custom.scss

```diff
@@ -270,7 +270,9 @@ footer {
 
 /* Styles for sections that are rendered from data, such as HTTP APIs and event schemas */
 .rendered-data {
-  margin: 1rem 0 3rem 0;
+  background-color: $secondary-lightest-background;
```

Comment:
This seems to have a jarring reaction with the arrow on the details section:

![image](https://user-images.githubusercontent.com/1190097/182490856-c9d9be52-5f86-4c80-a11a-1a491c57677f.png)


(accented for visibility)

Reply:
This can be fixed with `list-style-position: inside;` on the `summary` (rule just below here)